peer/connector.go

```diff
@@ -505,3 +599,21 @@ func (conn *connector) releseServerKey(serverPublicKey []byte) error {
 	})
 	return nil
 }
+
+func (c *connector) nextState() {
```

Comment:
I don't understand your idea when changing from `conn.state += 1` to `nextState()`. Is it unnecessary?

Reply:
I can know connection go to next state
Plus one I don't know the purpose, I have to read code

Reply:
Agree